src/IconButton/index.scss

```diff
@@ -0,0 +1,100 @@
+.nds-icon-button {
+  --button-radius: 50%;
+  background: none;
+  color: inherit;
+  border: none;
+  font: inherit;
+  cursor: pointer;
+  outline: inherit;
+  font-size: var(--font-size-default);
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  border-radius: var(--button-radius);
+  height: 32px;
+  width: 32px;
+  /* mobile buttons are larger for ease of tapping */
+  @media (max-width: #{map.get($breakpoints, "s")}) {
+    height: 48px;
+    width: 48px;
+  }
+
+  /* Nudge the large size icons to be centered more accurately */
+  &.fontSize--l {
+    [class^="narmi-icon-"] {
+      position: relative;
```

Comment:
would recommend translating instead, but not a huge deal to keep it this way

`transform: translate(-1px, -0.5px);`

Translate accomplishes the same thing without any performance hits if for some reason we want to animate hover or active states